Makefile

```diff
@@ -23,18 +25,12 @@
 check: shellcheck shfmtcheck
 
 package:
-	sed "s/@@VERSION@@/$$(date '+%Y.%m.%d.%H')/" \
-		debian/changelog.in > debian/changelog
-
-	dpkg-buildpackage
-
-	@for ext in dsc tar.xz; do \
-		mv -v ../delphix-platform_*.$$ext artifacts; \
-	done
-
-	@for ext in buildinfo changes deb; do \
-		mv -v ../delphix-platform_*_amd64.$$ext artifacts; \
-	done
+	rm -rf artifacts
+	mkdir artifacts
+	dch --create --package delphix-platform -v $(VERSION) \
+			"Automatically generated changelog entry."
+	dpkg-buildpackage -b -uc -us
+	mv ../delphix-platform*.deb artifacts/
```

Comment:
why don't we move the other files (i.e. the non-`.deb` files) into the artifacts directory anymore? It's not clear why the prior `mv` commands needed to be changed.

Reply:
`linux-pkg` doesn't care about the other deliverables since we do not consume them during the build. That said I've reverted back the logic to what it was for the independent build. `linux-pkg` just doesn't use the other artifacts.